content/blog/2021-02-18-dvc-2-0-pre-release.md

```diff
@@ -0,0 +1,562 @@
+[`--params` option](/doc/command-reference/run#for-displaying-and-comparing-data-science-experiments)
+of `dvc run` for more details.
+
+### Iterating over params with foreach stages
+
+Iterating over params was a frequently requested feature. Now users can define
+multiple similar stages with a templatized command.
+
+```yaml
+stages:
+  build:
+    foreach:
+      gb:
+        thresh: 15
+        filename: 'model-gb.hdf5'
+      us:
+        thresh: 10
+        filename: 'model-us.hdf5'
+    do:
+      cmd: >-
+        python script.py --out ${item.filename} --thresh ${item.thresh}
+      outs:
+        - ${item.filename}
+```
+
+## Lightweight ML experiments
+
+DVC uses Git as a foundation for ML experiments. This solid foundation makes
+each ML experiment reproducible and accessible from Git history. This Git-based
+approach works very well for ML projects with mature ML models when only a few
+new experiments per day are running. However, in more active development when
+dozens or hundreds of experiments need to be run in a single day, Git creates
+overhead - each experiment run requires additional Git commands
+`git add/commit`, and comparing all experiments is difficult.
+
+We introduce lightweight experiments in DVC 2.0! This is the way of
+auto-tracking without any overhead from ML engineers.
+
+⚠️ Note, ML experiment is an experimental feature in the coming release. It
+means the commands might change a bit even after the release.
+
+Run an ML experiment with a new hyperparameter from `params.yaml`:
+
+```dvc
+$ dvc exp run --set-param featurize.max_features=3000
+
+Reproduced experiment(s): exp-bb55c
+Experiment results have been applied to your workspace.
+
+$ dvc exp diff
+Path         Metric    Value    Change
+scores.json  auc       0.57462  0.0072197
+
+Path         Param                   Value    Change
+params.yaml  featurize.max_features  3000     1500
+```
+
+More experiments:
+
+```dvc
+$ dvc exp run --set-param featurize.max_features=4000
+Reproduced experiment(s): exp-9bf22
+Experiment results have been applied to your workspace.
+
+$ dvc exp run --set-param featurize.max_features=5000
+Reproduced experiment(s): exp-63ee0
+Experiment results have been applied to your workspace.
+
+$ dvc exp run --set-param featurize.max_features=5000 \
+                --set-param featurize.ngrams=3
+Reproduced experiment(s): exp-80655
+Experiment results have been applied to your workspace.
+```
+
+In the examples above, hyperparamters were changed automaticaly by option
+`--set-param`. User can make this changes manualy by modifying the file. The
+same way _any code or data files can be changed_ and `dvc exp run` will capture
+the changes.
+
+See all the runs:
+
+```dvc
+$ dvc exp show --no-pager --no-timestamp \
+        --include-params featurize.max_features,featurize.ngrams
+┏━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
+┃ Experiment    ┃     auc ┃ featurize.max_features ┃ featurize.ngrams ┃
```

Comment:
The second line looks like it has too many spaces after "Experiment" in the published version.